src/cpp/session/SessionOptions.cpp

```diff
@@ -707,6 +721,17 @@ void Options::resolveCopilotPath(const FilePath& resourcePath, std::string* pPat
    resolvePath(resourcePath, pPath);
 }
 
+void Options::resolveNodePath(const FilePath& resourcePath, std::string* pPath)
+{
+#if defined(__linux__) && !defined(RSTUDIO_PACKAGE_BUILD)
+   // node version should match RSTUDIO_INSTALLED_NODE_VERSION
+   FilePath dependenciesPath = resourcePath.completePath("../../dependencies/common/node/22.13.0-installed");
```

Comment:
Any chance we can have this version number flow in from CMake?

Reply:
Good idea, I'll look into doing that.